Dockerfile

```diff
@@ -5,7 +5,13 @@ COPY package-lock.json* .
 RUN npm ci
 
 FROM node:20-slim
+
+RUN apt update ; \
```

Comment:
we should use mount cache here. see https://docs.docker.com/build/guide/mounts/

Reply:
Let me investigate and add a patch based on what I can come up with.